src/main/resources/config/liquibase/changelog/20241013150000_changelog.xml

```diff
@@ -0,0 +1,67 @@
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <!-- Change Set for creating a new table for storing multiple public SSH keys of users -->
+    <changeSet id="20241013150000" author="entholzer">
+        <createTable tableName="user_public_ssh_key">
+            <column autoIncrement="true" name="id" type="bigint">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="user_id" type="bigint">
+                <constraints nullable="false"/>
+            </column>
+            <column name="label" type="VARCHAR(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="public_key" type="varchar(1000)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="key_hash" type="VARCHAR(100)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="creation_date" type="TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+            <column name="last_used_date" type="TIMESTAMP">
+                <constraints nullable="true"/>
+            </column>
+            <column name="expiry_date" type="TIMESTAMP">
+                <constraints nullable="true"/>
+            </column>
+        </createTable>
+        <createIndex indexName="idx_user_public_ssh_key_user_id" tableName="user_public_ssh_key">
+            <column name="user_id"/>
+        </createIndex>
+        <createIndex indexName="idx_user_public_ssh_key_ssh_public_key_hash" tableName="user_public_ssh_key">
+            <column name="key_hash"/>
+        </createIndex>
+    </changeSet>
+
+    <!-- Modify the id column for MySQL -->
+    <changeSet id="20241013150000_1" author="entholzer">
+        <preConditions onFail="MARK_RAN">
+            <dbms type="mysql"/>
+        </preConditions>
+        <modifyDataType tableName="user_public_ssh_key" columnName="id" newDataType="BIGINT AUTO_INCREMENT"/>
+    </changeSet>
+
+    <!-- Modify the id column for PostgreSQL -->
+    <changeSet id="20241013150000_2" author="entholzer">
+        <preConditions onFail="MARK_RAN">
+            <dbms type="postgresql"/>
+        </preConditions>
+        <modifyDataType tableName="user_public_ssh_key" columnName="id" newDataType="BIGSERIAL"/>
+    </changeSet>
+
+    <changeSet id="20241013150000_3" author="entholzer">
+        <sql>
+            INSERT INTO user_public_ssh_key (user_id, label, public_key, key_hash, creation_date, last_used_date, expiry_date)
+            SELECT id, 'Key 1', ssh_public_key, ssh_public_key_hash, CURRENT_TIMESTAMP, NULL, NULL
+            FROM jhi_user
+            WHERE ssh_public_key IS NOT NULL;
+        </sql>
+    </changeSet>
+
```

Comment:
one change set is missing that deletes the shh public key columns in the `jhi_user` table

Reply:
I kinda left the delete out on purpose for now, so SSH wouldn't break for other PRs using the same databases. I added a disclaimer at the top to only use one specific test server, so it stays usable for the other test servers then